### PR TITLE
Bws05

### PR DESCRIPTION
**Pull request by cmgustavo**
* setup pin page
* loading page
* active menu
* walletService
* receive section

### CODE REVIEW

File: src/js/services/walletService.js [hunk 1]
```diff
@@ -1,186 +1,69 @@
 'use strict';
```
Comment (matiu) [inline src/js/services/walletService.js]: Please lets not use walletService at all.

File: src/js/services/walletService.js [hunk 2]
```diff
@@ -1,186 +1,69 @@
...
+      profileService.focusedClient.createAddress(function(err, res) {
         return cb(err, res);
       });
     };
 
-    root.getAddresses = function(cb) {
-      var walletData = $rootScope.iden.wallets[$rootScope.currentWallet.wallet.id];
-
-      bwcService.getAddresses(walletData, function(err, res) {
+    root.getMainAddresses = function(cb) {
```
Comment (matiu) [inline src/js/services/walletService.js]: we dont need these functions. We can call directly getMainAddresses

File: src/js/services/walletService.js [hunk 3]
```diff
@@ -1,186 +1,69 @@
...
-          };
-          $rootScope.offline = true;
-        } else {
-          $rootScope.currentWallet = res;
-          $rootScope.offline = false;
-        }
-        return cb(err, res);
+    var updateStatus = function(cb) {
+      profileService.focusedClient.getStatus(function(err, walletStatus) {
+        $rootScope.$emit('newFocusedWallet', walletStatus);
```
Comment (matiu) [inline src/js/services/walletService.js]: no, please do not reuse event names, with names that are not related

**Comment by cmgustavo** [comment]
I got it. I updated my PR: removes walletService to avoid confusion and uses the `focusedClient` to get new address.
